all: build fails on darwin

### Describe the bug

Building things that use this library seems to yield an error on darwin, both amd64 and arm64.

Example output:

```
❯ GOOS=darwin GOARCH=arm64 go build .
# github.com/mattn/go-ieproxy
../../Go/pkg/mod/github.com/mattn/go-ieproxy@v0.0.3/ieproxy.go:36:9: undefined: getConf
../../Go/pkg/mod/github.com/mattn/go-ieproxy@v0.0.3/ieproxy.go:41:9: undefined: reloadConf
../../Go/pkg/mod/github.com/mattn/go-ieproxy@v0.0.3/ieproxy.go:48:2: undefined: overrideEnvWithStaticProxy
../../Go/pkg/mod/github.com/mattn/go-ieproxy@v0.0.3/ieproxy.go:53:13: psc.findProxyForURL undefined (type *ProxyScriptConf has no field or method findProxyForURL, but does have FindProxyForURL)
```

### To Reproduce

```
GOOS=darwin GOARCH=arm64 go build .
```

## Expected behavior

Build just fine.

### Version

0.25.

### Additional context

v0.24 works fine. It seems that its related to the update of go-ieproxy to v0.0.3.

--- a/go.mod
+++ b/go.mod
@@ -64,7 +64,6 @@ require (
 	github.com/googleapis/gax-go/v2 v2.2.0
 	github.com/klauspost/compress v1.15.1 // indirect
 	github.com/lib/pq v1.10.4
-	github.com/mattn/go-ieproxy v0.0.3 // indirect
 	github.com/mitchellh/mapstructure v1.4.3 // indirect
 	go.opencensus.io v0.23.0
 	go.uber.org/atomic v1.9.0 // indirect

